apps/api/src/app/events/usecases/process-bulk-cancel/process-bulk-cancel.usecase.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common';
+import { TriggerBulkCancelResponseDto } from '../../dtos/trigger-bulk-cancel-response.dto';
+import { CancelDelayed, CancelDelayedCommand } from '../cancel-delayed';
+import { ProcessBulkCancelCommand } from './process-bulk-cancel.command';
+
+@Injectable()
+export class ProcessBulkCancel {
+  constructor(private cancelDelayed: CancelDelayed) {}
+
+  async execute(command: ProcessBulkCancelCommand) {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add a return typing to this function using the Dto to fully enforce the use-case boundary with type-safety.‏

Reply:
done